pulp_2_tests/tests/rpm/api_v2/test_updateinfo.py

```diff
@@ -200,28 +202,29 @@ def do_test(self, date):
         return [
             item.find('updated').get('date')
             for item in update_elements.findall('update')
+            if item.find('id').text == self.errata['id']
         ]
 
     def test_01_valid_date(self):
-        """Update errata using a valid date. See :meth:`do_test`."""
+        """Update errata using a valid date. See:meth:`do_test`."""
         updates = self.do_test(ERRATA_UPDATE_INFO['updated_date'])
         self.assertEqual(len(updates), 1, updates)
         self.assertEqual(updates[0], ERRATA_UPDATE_INFO['updated_date'])
 
     def test_02_older_date(self):
-        """Update errata using a valid older date. See :meth:`do_test`."""
+        """Update errata using a valid older date. See:meth:`do_test`."""
         updates = self.do_test(ERRATA_UPDATE_INFO['old_updated_date'])
         self.assertEqual(len(updates), 1, updates)
         self.assertEqual(updates[0], ERRATA_UPDATE_INFO['updated_date'])
 
     def test_03_newer_date(self):
-        """Create errata using a newer valid date. See :meth:`do_test`."""
+        """Create errata using a newer valid date. See:meth:`do_test`."""
         updates = self.do_test(ERRATA_UPDATE_INFO['new_updated_date'])
         self.assertEqual(len(updates), 1, updates)
         self.assertEqual(updates[0], ERRATA_UPDATE_INFO['new_updated_date'])
 
     def test_04_invalid_date(self):
-        """Attempt to upload an errata with an invalid date."""
+        """Attemp to upload an errata with an invalid date."""
```

Comment:
Those docstrings changes are already present on the code basis. [docs](https://pulp-2-tests.readthedocs.io/en/latest/tests/pulp_2_tests.tests.rpm.api_v2.test_updateinfo.html#pulp_2_tests.tests.rpm.api_v2.test_updateinfo.DateUpdateErrataTestCase)